src/components/Thread.vue

```diff
@@ -94,8 +130,43 @@ export default {
 	},
 	created() {
 		this.resetThread()
+		window.addEventListener('resize', this.updateParticipantsToDisplay)
```

Comment:
should we debounce this a bit? I'm afraid a browser will actually fire this event many times as the user resizes. And so for every pixel this is recalculated

Reply:
My tests with FF latest didn't show so much events fired... Could you test a little too?

Reply:
Let's please add the debounce in any case. We can't test this with every possible browser. You can use https://lodash.com/docs/4.17.15#debounce for this.

Reply:
debounced with a 500ms delay